content-repo/extra-docs/articles/long-running-invoke.md

```diff
@@ -0,0 +1,57 @@
+---
+id: long-running-invoke
+title: Invoking Long Running HTTP Integrations via Server's HTTPS endpoint
+description: Explains how to set up long running integrations which expose an HTTP endpoint so they can be accessed via the Cortex XSOAR's HTTPS endpoint. 
+---
+
+## Overview
+**Supported Cortex XSOAR Server version**: 5.5 and above
+
+Cortex XSOAR supports setting up long running integrations which expose an HTTP endpoint. Such integrations include:
+* Palo Alto Networks PAN-OS EDL Service
+* Export Indicators Service
+* Microsoft Teams
+
+When you initiate these integrations, they listen on an incoming HTTP port. The port is configured via the **Listen Port** setting of the integration. The HTTP interface can be accessed directly over the port, for example by running curl locally on the Cortex XSOAR Server machine (assuming the configured port is 7000 and HTTP is being used):
```

Comment:
we hide the port parameter in the integration, so might worth to mention it here so people wont go and look for it